pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: awesome_notifications
 description: A complete solution to create Local Notifications and Push Notifications (Media Notifications, Big Picture, Big text, etc), through Firebase or another services, using Flutter.
-version: 0.7.0-beta.2
+version: 0.7.0-beta.4
```

Comment:
the next beta version is 3 or I'm wrong?

Reply:
because of my previous commit